### ISSUE TITLE
sbt dist generates -sans-externalized.jar without manifest entries

### ISSUE THREAD

**Issue by elektro-wolle** [playframework/playframework#7010]
### Play Version (2.5.x / etc)
play-2.5

### API (Scala / Java / Neither / Both)
scala

### Expected Behavior

Please describe the expected behavior of the issue, starting from the first action.

1. build play scala project with `sbt dist`
2. the `target/$name-$version.jar` contains a `META-INF/MANIFEST.MF` with e.g. `Implementation-Version`, `Implementation-Vendor`
3. the `target/$name-$version-sans-externalized.jar` in the resulting `target/universal/...zip` file should contain the same Manifest entries

### Actual Behavior

1.  the `target/$name-$version-sans-externalized.jar` in the resulting zip file contains only the `Manifest-Version`

**Comment by marcospereira** [comment]
See #4522 and this discussion:

https://groups.google.com/forum/#!topic/play-framework/hePH94MPQk8

Is that what you want?

**Comment by elektro-wolle** [comment]
Unfortunately not. I'ld like to have the same manifest.mf entries in the resulting project jar file, regardless of the setting of `PlayKeys.externalizeResources`.

With `PlayKeys.externalizeResources := false`:
```
$ sbt clean dist
$ unzip -p target/project-1.1.1-SNAPSHOT.jar META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: project
Implementation-Version: 1.1.1-SNAPSHOT
Specification-Vendor: org.example
Specification-Title: project
Implementation-Vendor-Id: org.example
Specification-Version: 1.1.1-SNAPSHOT
Implementation-Vendor: org.example
Main-Class: play.core.server.ProdServerStart
```
produces as expected:
```
$ cd target/universal/
$ unzip project-1.1.1-SNAPSHOT.zip
$ unzip -p  project-1.1.1-SNAPSHOT/lib/org.example.project-1.1.1-SNAPSHOT.jar META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: project
Implementation-Version: 1.1.1-SNAPSHOT
Specification-Vendor: org.example
Specification-Title: project
Implementation-Vendor-Id: org.example
Specification-Version: 1.1.1-SNAPSHOT
Implementation-Vendor: org.example
Main-Class: play.core.server.ProdServerStart
```

with `PlayKeys.externalizeResources := true`, the manifest entries were missing in the resulting project jar:
```
$ cd target/universal/
$ unzip project-1.1.1-SNAPSHOT.zip
$ unzip -p project-1.1.1-SNAPSHOT/lib/org.example.project-1.1.1-SNAPSHOT-sans-externalized.jar META-INF/MANIFEST.MF
Manifest-Version: 1.0
```

**Comment by mkurz** [comment]
This is fixed via #12739 so all generated jars now have the same manifest file. Be aware however that the `Main-Class` entry got removed, because it did not make any sense.
